frontend/src/modules/LoadTestContentRedux.js

```diff
@@ -16,9 +17,27 @@ const getTestMetaData = testName => {
   };
 };
 
+const updateTestInstructionsState = testInstructions => ({
+  type: UPDATE_TEST_INSTRUCTIONS,
+  testInstructions
+});
+
+const getTestInstructions = testName => {
+  return async function() {
+    let testInstructionsContent = await fetch(`/api/test-instructions/?test_name=${testName}`, {
+      method: "GET",
+      Accept: "application/json",
+      "Content-Type": "application/json",
+      cache: "default"
+    });
+    return await testInstructionsContent.json();
+  };
+};
+
 // Initial State
 const initialState = {
-  testMetaData: ""
+  testMetaData: "",
+  testInstructions: ""
```

Comment:
Initial state should be {}

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/355/commits/f3d35e26a1423071095293f28ec84c9888e16c55